.github/workflows/test.yml

```
@@ -0,0 +1,213 @@
+              run: |
+          if [ -f "packages/adapter-appwrite/package.json" ]; then
+            cd packages/adapter-appwrite
+            if grep -q '"typecheck"' package.json; then
+              pnpm run typecheck
+            else
+              echo "No typecheck script found for adapter-appwrite"
+            fi
+          else
+            echo "adapter-appwrite package not found"
+          fi
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)